agreement/views/agreement.xml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" ?>
+<!--
+  © 2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+
+
+<record id="agreement_form" model="ir.ui.view">
+    <field name="name">agreement.form</field>
+    <field name="model">agreement</field>
+    <field name="arch" type="xml">
+        <form string="Agreement">
+            <sheet>
```

Comment:
Please add 
`<div class="oe_button_box" name="button_box"/>` so other modules can add smartbuttons to main view

Reply:
done